misc/store_np_arrays_binary.py

```diff
@@ -22,7 +22,7 @@
 import time
 import numpy as np
 
-N = 50000
+N = 50000000
 fname = '/tmp/np_array_test.bin'
```

Comment:
I've been using "/tmp" out of lazyness. There is a python function to get the system tmp directory: https://stackoverflow.com/questions/847850/cross-platform-way-of-getting-temp-directory-in-python